frontend-react/src/components/USLink.tsx

```diff
@@ -53,6 +54,52 @@ export const USLink = ({
     );
 };
 
+export interface USLinkButtonProps
+    extends USLinkProps,
+        Omit<ButtonProps, "type"> {}
+
+export const USLinkButton = ({
+    anchor = false,
+    children,
+    className,
+    href,
+    state,
+    secondary,
+    accentStyle,
+    base,
+    outline,
+    inverse,
+    size,
+    unstyled,
+    ...anchorHTMLAttributes
+}: USLinkButtonProps) => {
+    const linkClassname = classnames(
```

Comment:
`classnames` can accept an object so we don't have to use all these ternaries:

```jsx
classnames("usa-button", {
    "usa-button--secondary": secondary,
    [`usa-button--accent-${accentStyle}`]: accentStyle,
    ....
}, className);
```

Reply:
I knew there was a cleaner way to do it, but couldn't remember. Ty!